popup.css

```diff
@@ -83,6 +83,27 @@
   gap: 10px;
 }
 
+.btnn {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 7px 0;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  background-color: #fff;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 13px;
+  line-height: 24px;
+  color: #396df2;
+  cursor: pointer;
+}
+
+a {
+  text-decoration: none;
+}
+
 @media only screen and (min-width: 768px) {
   .project1 {
     background-color: #fff;
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/224512867-eca03bfb-e672-486f-a500-ddd7387604fe.png)

**[MANDATORY CHANGE]**

- [ ] As suggested by the previous reviewer, it would be nice if you make your card content close to the right and left edges to follow the Figma template 😊 You can see the highlighted section with red pen 🙄 Kindly reduce that 👍